### PR TITLE
RTC - Added support for specifying parent work item

### PR DESCRIPTION
**Pull request by sibartlett**

### CODE REVIEW

File: sdetools/modules/sync_rational/rational_plugin.py [hunk 1]
```diff
@@ -293,6 +293,15 @@ def _get_rtc_field(self, field_name, field_value):
         if not field_name:
             raise AlmException('Unsupported attribute: %s' % attribute)
 
+        if rs['oslc:name'] == 'com.ibm.team.workitem.linktype.parentworkitem.parent':
```
Comment (geoffwhittington) [inline sdetools/modules/sync_rational/rational_plugin.py]: This looks good. Better to not match against "Parent"

File: sdetools/modules/sync_rational/rational_plugin.py [hunk 2]
```diff
@@ -293,6 +293,15 @@ def _get_rtc_field(self, field_name, field_value):
...
+            resource_url = self.alm_plugin.get_url('resource/itemName/com.ibm.team.workitem.WorkItem/%s' % field_value)
```
Comment (geoffwhittington) [inline sdetools/modules/sync_rational/rational_plugin.py]: What is an example of "field_value"?

Comment (geoffwhittington) [inline sdetools/modules/sync_rational/rational_plugin.py]: I am only concerned that `field_value` is an internal value and not a user-friendly one. The use case is developers will be entering the name of the Epic and I'm expecting that value should be obvious from the RTC UI

Comment (sibartlett) [inline sdetools/modules/sync_rational/rational_plugin.py]: It's the numerical workitem ID, which is displayed next to the name of the Epic in RTC UI.

**Pull request merged**